src/components/MyButton.js

```diff
@@ -0,0 +1,11 @@
+
+import React from "react";
+
+// const MyButton = props => <button className="btn btn-primary fn-submit-name">Search IDs</button>
+// const MyButton = props => <button className="btn btn-primary fn-submit-name">{props.text}</button>
```

Comment:
In general its considered a bad idea to leave commented out code in the code base.